tools/osbuild-image-test

```diff
@@ -12,9 +12,11 @@ import subprocess
 import sys
 import traceback
 import shutil
+import re
+import tempfile
 
 from collections import defaultdict
-from typing import Iterator, List, Optional, Dict
+from typing import Iterator, List, Optional, Dict, final
```

Comment:
What is the `final` for? It does not seem to exist in the `typing` module and also it does not seem to be used anywhere in the code.

Reply:
I have no idea actually! Maybe my editor added it by mistake. I don't remember adding it.